{"uid":"41f9d96714b4df0e","name":"Тест логина пользователя с неверным паролем","fullName":"tests.api.test_auth#test_login_user_with_incorrect_password","historyId":"1d2a7af65c6652b24336a821deb0a4e2","time":{"start":1755681234923,"stop":1755681234923,"duration":0},"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 1 validation error for RegisterUserResponseSchema\ncreatedAt\n  Value error, Некорректный формат даты и времени [type=value_error, input_value='2025-08-20T09:13:56.033Z', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.11/v/value_error","statusTrace":"auth_requester = <custom_requester.custom_requester.CustomRequester object at 0x7ff3c048dd50>\ntest_user = UserSchema(email='ghkq7df4@gmail.com', fullName='Kelly Bruce', password='j#5&69vrUwc$LH?6QSA', passwordRepeat='j#5&69vrUwc$LH?6QSA', roles=[<Roles.USER: 'USER'>], banned=None, verified=None)\n\n    @pytest.fixture(scope=\"function\")\n    def registered_user(auth_requester, test_user: UserSchema) -> UserFixtureSchema:\n        \"\"\"\n        Фикстура для регистрации и получения данных зарегистрированного пользователя.\n        \"\"\"\n        response = auth_requester.send_request(\n            method=\"POST\",\n            endpoint=REGISTER_ENDPOINT,\n            data=test_user.model_dump(),\n            expected_status=201\n        )\n        return UserFixtureSchema(\n            request=test_user,\n>           response=RegisterUserResponseSchema.model_validate_json(response.text))\nE       pydantic_core._pydantic_core.ValidationError: 1 validation error for RegisterUserResponseSchema\nE       createdAt\nE         Value error, Некорректный формат даты и времени [type=value_error, input_value='2025-08-20T09:13:56.033Z', input_type=str]\nE           For further information visit https://errors.pydantic.dev/2.11/v/value_error\n\nconftest.py:120: ValidationError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1755681206191,"stop":1755681206192,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"delete_output_dir","time":{"start":1755681206192,"stop":1755681206192,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"pytestconfig","time":{"start":1755681206192,"stop":1755681206192,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1755681206192,"stop":1755681206192,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1755681206193,"stop":1755681206193,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"auth_requester","time":{"start":1755681233786,"stop":1755681233786,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"user_session","time":{"start":1755681234923,"stop":1755681234923,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"admin","time":{"start":1755681234924,"stop":1755681235768,"duration":844},"status":"passed","steps":[{"name":"Аутентификация сессии","time":{"start":1755681234924,"stop":1755681235768,"duration":844},"status":"passed","steps":[{"name":"Логин пользователя /login","time":{"start":1755681234924,"stop":1755681235767,"duration":843},"status":"passed","steps":[{"name":"Отправка 'POST'-запроса на '/login'","time":{"start":1755681234924,"stop":1755681235767,"duration":843},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"method","value":"'POST'"},{"name":"endpoint","value":"'/login'"},{"name":"data","value":"LoginRequestSchema(email='api1@gmail.com', password='asdqwe123Q')"},{"name":"expected_status","value":"200"},{"name":"need_logging","value":"True"},{"name":"params","value":"None"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"login_data","value":"LoginRequestSchema(email='api1@gmail.com', password='asdqwe123Q')"},{"name":"expected_status","value":"200"}],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"user_creds","value":"LoginRequestSchema(email='api1@gmail.com', password='asdqwe123Q')"}],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":3,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"test_user","time":{"start":1755681235768,"stop":1755681235768,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"registered_user","time":{"start":1755681235768,"stop":1755681236093,"duration":325},"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 1 validation error for RegisterUserResponseSchema\ncreatedAt\n  Value error, Некорректный формат даты и времени [type=value_error, input_value='2025-08-20T09:13:56.033Z', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.11/v/value_error\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Cinescope_wraithmainer/Cinescope_wraithmainer/conftest.py\", line 120, in registered_user\n    response=RegisterUserResponseSchema.model_validate_json(response.text))\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pydantic/main.py\", line 746, in model_validate_json\n    return cls.__pydantic_validator__.validate_json(\n","steps":[{"name":"Отправка 'POST'-запроса на '/register'","time":{"start":1755681235769,"stop":1755681236093,"duration":324},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"method","value":"'POST'"},{"name":"endpoint","value":"'/register'"},{"name":"data","value":"{'email': 'ghkq7df4@gmail.com', 'fullName': 'Kelly Bruce', 'password': 'j#5&69vrUwc$LH?6QSA', 'passwordRepeat': 'j#5&69vrUwc$LH?6QSA', 'roles': [<Roles.USER: 'USER'>], 'banned': None, 'verified': None}"},{"name":"expected_status","value":"201"},{"name":"need_logging","value":"True"},{"name":"params","value":"None"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 1 validation error for RegisterUserResponseSchema\ncreatedAt\n  Value error, Некорректный формат даты и времени [type=value_error, input_value='2025-08-20T09:13:56.033Z', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.11/v/value_error","statusTrace":"auth_requester = <custom_requester.custom_requester.CustomRequester object at 0x7ff3c048dd50>\ntest_user = UserSchema(email='ghkq7df4@gmail.com', fullName='Kelly Bruce', password='j#5&69vrUwc$LH?6QSA', passwordRepeat='j#5&69vrUwc$LH?6QSA', roles=[<Roles.USER: 'USER'>], banned=None, verified=None)\n\n    @pytest.fixture(scope=\"function\")\n    def registered_user(auth_requester, test_user: UserSchema) -> UserFixtureSchema:\n        \"\"\"\n        Фикстура для регистрации и получения данных зарегистрированного пользователя.\n        \"\"\"\n        response = auth_requester.send_request(\n            method=\"POST\",\n            endpoint=REGISTER_ENDPOINT,\n            data=test_user.model_dump(),\n            expected_status=201\n        )\n        return UserFixtureSchema(\n            request=test_user,\n>           response=RegisterUserResponseSchema.model_validate_json(response.text))\nE       pydantic_core._pydantic_core.ValidationError: 1 validation error for RegisterUserResponseSchema\nE       createdAt\nE         Value error, Некорректный формат даты и времени [type=value_error, input_value='2025-08-20T09:13:56.033Z', input_type=str]\nE           For further information visit https://errors.pydantic.dev/2.11/v/value_error\n\nconftest.py:120: ValidationError","steps":[],"attachments":[{"uid":"e91d2b0ec4dfecc6","name":"log","source":"e91d2b0ec4dfecc6.txt","type":"text/plain","size":2736}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"user_session::0","time":{"start":1755681236102,"stop":1755681236102,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests.api"},{"name":"suite","value":"test_auth"},{"name":"host","value":"pkrvmubgrv54qmi"},{"name":"thread","value":"3917-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.api.test_auth"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"41f9d96714b4df0e.json","parameterValues":[]}